### PR TITLE
Add a convenience method to turn a Position into Placemarks.

### PR DESCRIPTION
**Pull request by Lootwig**

### CODE REVIEW

File: lib/geolocator.dart [hunk 1]
```diff
@@ -9,10 +9,15 @@ import 'package:meta/meta.dart';
 import 'package:permission_handler/permission_handler.dart';
 
 part 'models/geolocation_enums.dart';
+
```
Comment (mvanbeusekom) [inline lib/geolocator.dart]: Please remove the empty line

File: lib/geolocator.dart [hunk 2]
```diff
@@ -9,10 +9,15 @@ import 'package:meta/meta.dart';
...
 part 'models/location_accuracy.dart';
+
```
Comment (mvanbeusekom) [inline lib/geolocator.dart]: Please remove the empty line

File: lib/geolocator.dart [hunk 3]
```diff
@@ -9,10 +9,15 @@ import 'package:meta/meta.dart';
...
 part 'models/location_options.dart';
+
```
Comment (mvanbeusekom) [inline lib/geolocator.dart]: Please remove the empty line

File: lib/geolocator.dart [hunk 4]
```diff
@@ -9,10 +9,15 @@ import 'package:meta/meta.dart';
...
 part 'models/placemark.dart';
+
```
Comment (mvanbeusekom) [inline lib/geolocator.dart]: Please remove the empty line

File: lib/geolocator.dart [hunk 5]
```diff
@@ -9,10 +9,15 @@ import 'package:meta/meta.dart';
...
 part 'models/position.dart';
+
```
Comment (mvanbeusekom) [inline lib/geolocator.dart]: Please remove the empty line

File: lib/geolocator.dart (hunk 1 above)
Comment (Lootwig) [reply lib/geolocator.dart]: I was wondering about these. My IDE is set to use dartfmt for formatting and I made no customizations of the style, do you have any clue about fixing this (using IntelliJ)?

Comment (mvanbeusekom) [reply lib/geolocator.dart]: That is strange, I always run `flutter format .` in the root of the project (which to my knowledge is an alias of dartfmt) and for me it removes the empty lines. 

Maybe `flutter format .` is more then a simple alias and has a bit different rule set compared to the default dartfmt?

**Pull request merged**